docs/Microservices/RPS/securityRPS.md

```diff
@@ -2,13 +2,14 @@
 
 # RPS Security Considerations
 
-Remote Provision Service (RPS) is built to be a micro service that plays a component role in a larger set of services that makes up the device management software suite.  In this role, RPS uses and creates secrets that are required to be able to successfully activate and use Intel® AMT.  There are five key assets that must be protected:
+Remote Provision Service (RPS) is built to be a micro service that plays a component role in a larger set of services that makes up the device management software suite.  In this role, RPS uses and creates secrets that are required to be able to successfully activate and use Intel® AMT.  There are six key assets that must be protected:
```

Comment:
Isn't microservice one word? Also you can maybe get rid of "is built to be" if you just do this: 
The microservice Remote Provision Service (RPS) plays a component role in a larger set of services that makes up the device management software suite. 

Or something like that. I leave it to you.

Reply:
Updated